### DISCUSSION TITLE
About the Chinese install video

### DISCUSSION THREAD

**Discussion by KNA7022** [SnailyCAD/snaily-cadv4#764] [Q&A] [answered]
If I'm on www.bilibili.com upload an installation tutorial video in Chinese that talks about how some mistakes happened and, of course, also how they were installed and can you place the link on your site?

**Comment by casperiv0** [comment, marked as answer]
sure, a link will be placed on the documentation site.

**Reply by KNA7022** [reply]
ok coming soon thanks
